### DISCUSSION TITLE
Need new instructions regarding tab issues

### DISCUSSION THREAD

**Discussion by KathrynBassett** [Aris-t2/CustomCSSforFx#557] [Q&A] [answered]
Background to my problem: I have Firefox 110.0 "up to date" (Win 10 if that makes a difference). Over the past several months I've been having more and more problems with tabs. The the only visible tab would the one currently in use and to get to another one I'd have to use control-1 or control-2, whatever tab number was the one I think I want; I couldn't tell except by guessing. Another issue would be that the find bar would be in the same spot as the first tab. So if I was actually on the first tab, I couldn't read what was in the find bar. But, just recently, and I assume it's when FF went to 110, the tabs (at bottom) have disappeared entirely. I have no idea how many tabs are open. And since there are no tabs, I can't click on the tab's X to close it. I have to use control-w to close the tab.

So, I decided that it's time to update Aris-t2 CustomCSSforFx since it's been awhile. I downloaded custom_css_for_fx_v4.3.1v7.zip, and made a new profile. Now I'm on to makeing the changes in my userChrome.css. The last time I did this, the approximate line numbers I needed to edit were 
360: @import "./css/tabs/tabs_below_navigation_toolbar_fx74.css"; /*Kathryn uncommented*/
646: @import "./css/toolbars/bookmarks_toolbar_multiple_lines_fx74.css"; /*Kathryn uncommented*/ /* <--- EXPERIMENTAL */ /*Fx74+ */

Those two css files no longer exhist so I need new instructions on what to modify to get the tabs as the bottom. After that, I will also need to tackle having 4 lines of bookmarks instead of the default 3 and getting all the bookmarks from my current profile to my new profile.

**Comment by MarkRH** [comment, marked as answer]
Repeating what I said in issues:

Well, my guess would be that the first one is line **571**:
`/* @import "./css/tabs/tabs_below_navigation_toolbar.css"; /**/`

The other one is line **1052**:
`/* @import "./css/toolbars/bookmarks_toolbar_multiple_lines.css"; /**/`

Hope that helps.

**Comment by KathrynBassett** [comment]
Finally had a chance to work on your reply (for which I thank you). These worked, then I went on to part two regarding the tabs. What I had actually wanted was line 608 to put the tabs at the bottom of the main content vs bottom of the bookmarks. But since your answer got me in the right area, it was then just a matter of reading the (well documented) comments to find the correct tabs section. I'm now all set.

**Comment by MarkRH** [comment]
Ahh.. OK. Glad you got it working. :)

**Comment by Aris-t2** [comment]
Please do not start new discussions. Use the one already available: https://github.com/Aris-t2/CustomCSSforFx/discussions/454
